releases/handbook.md

```diff
@@ -194,15 +193,38 @@ Criteria for timeline that the team needs to consider
 - Kubecon dates - let’s not hard block on events, but keep them in mind since we know community members might get doublebooked.
 - Associated events (aka. AI Day at Kubecon, Tensorflow events) - we want to keep them in mind.
 
-**Success Criteria:** Release team selected, schedule sent to kubeflow-discuss, all release team members have the proper permissions and are meeting regularly.
+**Success Criteria:**
+* Release team selected
+* Schedule sent to kubeflow-discuss
```

Comment:
Can we rephrase this to "send out release timeline proposal PR to kubeflow-discuss to receive feedback from the community"? and maybe during the week 1 phase, we have success criteria of "Send out the finalized release schedule to kubeflow-discuss"?

Reply:
Since there are the following checkboxes in the [preparation phase](https://github.com/kubeflow/community/blob/master/releases/handbook.md#preparation) above the success criteria:
> [ ] Publish draft schedule to kubeflow-discuss, with actual dates
> [ ] Get lazy consensus on the release schedule from the WG leads

WDYT about just replacing `Schedule sent to kubeflow-discuss` with your proposed `Send out the finalized release schedule to kubeflow-discuss` sentence there?

Currently the Preparation phase does not have a deadline, but from your comment about week 1 I understand you'd like to start defining this phase more concretely?

Reply:
Ah okay, if it's already in the preparation phase I think it's okay to leave it as it is. No need to rephrase. I wanted to make sure that the release timeline was proposed to the community for feedback before finalizing. :)